coreMacros/src/main/scala/chisel3/internal/sourceinfo/NamingAnnotations.scala

```diff
@@ -118,12 +118,14 @@ class NamingTransforms(val c: Context) {
   def transformModuleBody(stats: List[c.Tree]): Tree = {
```

Comment:
Done. I didn't see any comments to be updated in `Namer.scala` though since the case of Module is described as a special behaviour. 
Closing context on a non-module is not an issue.  
The only change to `Namer.scala` I would suggest is to rename `namePrefix` as `closeContext` as it sounds a bit clearer to me.